spec/services/github_issues_spec.rb

```diff
@@ -696,4 +696,172 @@
       end
     end
   end
+  
+  # These are integration tests after fashion, playing out
+  # some ping-pong interactions between Aha! and GitHub.
+  # I've isolated their setup from the tests above so they
+  # are resilient to breakage.
+  
+  context "when sending an existing feature to GitHub" do
+    
+    let(:service) do
+      AhaServices::GithubIssues.new(
+        'server_url' => "https://api.github.com",
+        'username' => 'user', 'password' => 'secret',
+        'repository' => 'user/repo'
+      )
+    end
+    
+    let(:api_client) { double }
+    let(:repo_resource) { double }
+    let(:milestone_resource) { double }
+    let(:issue_resource) { double }
+    let(:label_resource) { double }
+
+    before do
+      service.stub(:api).and_return(api_client)
+      service.stub(:repo_resource).and_return(repo_resource)
+      service.stub(:milestone_resource).and_return(milestone_resource)
+      service.stub(:issue_resource).and_return(issue_resource)
+      service.stub(:label_resource).and_return(label_resource)
+    end
+    
+    let(:aha_feature) do
+      Hashie::Mash.new(
+        name: 'Existing Feature',
+        workflow_status: workflow_status,
+        description: { body: 'Existing feature description' },
+        release: { name: 'Existing Feature Release' },
+        tags: [],
+        requirements: [],
```

Comment:
Style question: Are we okay with trailing commas?

Reply:
We generally prefer them, to get cleaners diffs when adding/removing items from data structures where we've chosen to put each item on its own line.